modules/xfem/include/base/XFEM.h

```diff
@@ -18,6 +18,7 @@
 #include "libmesh/quadrature.h"
 
 #include "GeometricCutUserObject.h"
+#include "LevelSetCutUserObject.h"
```

Comment:
Do we have to put `level set` in XFEM? I understand that you need `level set` to tell you the material belongs to which side. Can we make it not be too specific to `level set`? i.e. `mesh cut uo`.

Reply:
Yeah I'll talk to you to see how we should design this.

Reply:
Now this should be general enough to handle all geometric cuts. The related method is called `getConsistentSideBoolean` in the `GeometricCutUserObject` base class, which should be overridden by child classes. If it is not overridden AND healing is turned on AND there are stateful properties, it will throw an error.